/ros2_kdl_node:
  ros__parameters:
    cmd_interface: "velocity" 
    traj_type: "linear"
    s_type: "trapezoidal"

    traj_duration: 1.5
    acc_duration: 0.5

    total_time: 1.5
    trajectory_len: 150
    Kp: 5

    end_position_x: 0.3
    end_position_y: -0.2
    end_position_z: 0.3

    # Lambda scaling factor for joint limit avoidance
    lambda: 1.4